src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,71 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+import { PersonConnection } from '../PersonConnection/Person.Connection';
+
+type PeopleTableProps = {
+  people: Person[] | undefined;
+};
+
+export const PeopleTable = ({ people }: PeopleTableProps) => {
+  const { slugId } = useParams<{ slugId: string }>();
+  const selectedPerson = people?.find(person => person.slug === slugId);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+      <tbody>
+        {people?.map(person => {
```

Comment:
If `people` is undefined, the map function will not execute. Ensure that `people` is always defined when this component is rendered, or handle the case where it might be undefined to prevent potential runtime errors.